README.md

```diff
@@ -96,7 +96,9 @@ $ cd gateway
 
 Install dependencies:
 
-NOTE: `yarn` is preferred but npm install will also work.
+NOTE: `yarn` is preferred but `npm install` will also work. To install `yarn`,
+run `sudo apt-get install yarn` or follow the directions for your OS
+[here](https://yarnpkg.com/en/docs/install).
```

Comment:
nit: I'd probably make the 'directions for your OS' be the text for the URL rather than 'here', but that's just me.